ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt

```diff
@@ -38,41 +32,78 @@ import com.example.android.wearable.composeadvanced.R
 fun UserInputComponentsScreen(
     value: Int,
     onClickStepper: () -> Unit,
-    onClickSlider: () -> Unit
+    onClickSlider: () -> Unit,
+    onClickDemoDatePicker: () -> Unit,
+    onClickDemo12hTimePicker: () -> Unit,
+    onClickDemo24hTimePicker: () -> Unit,
 ) {
-    Column(
+    ScalingLazyColumn(
         modifier = Modifier
-            .fillMaxSize()
-            .padding(vertical = 10.dp)
-            .padding(horizontal = 10.dp),
-        verticalArrangement = Arrangement.spacedBy(4.dp, Alignment.CenterVertically)
+            .fillMaxSize(),
     ) {
-        Text(
-            modifier = Modifier.fillMaxWidth(1f),
-            text = "${stringResource(R.string.selected_value)}: $value",
-            textAlign = TextAlign.Center
-        )
+        item {
+            CompactChip(
+                onClick = onClickStepper,
+                label = {
+                    Text(
+                        stringResource(R.string.stepper_label, value),
+                        maxLines = 1,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                }
+            )
+        }
 
-        CompactChip(
-            onClick = onClickStepper,
-            label = {
-                Text(
-                    stringResource(R.string.stepper_label),
-                    maxLines = 1,
-                    overflow = TextOverflow.Ellipsis
-                )
-            }
-        )
+        item {
+            CompactChip(
+                onClick = onClickSlider,
+                label = {
+                    Text(
+                        stringResource(R.string.slider_label, value),
+                        maxLines = 1,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                }
+            )
+        }
 
-        CompactChip(
-            onClick = onClickSlider,
-            label = {
-                Text(
-                    stringResource(R.string.slider_label),
-                    maxLines = 1,
-                    overflow = TextOverflow.Ellipsis
-                )
-            }
-        )
+        item {
+            CompactChip(
+                onClick = onClickDemoDatePicker,
+                label = {
+                    Text(
```

Comment:
I would add the current value as a secondaryLabel on a Chip - ditto for next two